iron-overlay-behavior.html

```diff
@@ -308,28 +312,25 @@
 
       this._manager.addOrRemoveOverlay(this);
 
+      // TODO handle case when it was already animating, e.g. quick open/close.
```

Comment:
Include some signature next to your TODO, e.g., `// TODO(cdata):` - this way someone who reads this in the future can hunt you down and shame you :smiling_imp: